src/components/Reviews.test.jsx

```diff
@@ -0,0 +1,36 @@
+import { render } from '@testing-library/react';
+
+import ReviewItems from './Reviews';
+
+describe('Reviews', () => {
+  context('with reviews', () => {
+    it('renders reviews', () => {
+      const reviews = [
+        {
+          description: '훌륭하다 훌륭하다 지구인놈들',
+          id: 1,
+          name: '테스터',
+          restaurantId: 1,
+          score: 5,
+        },
+      ];
+
+      const { container } = render(<ReviewItems reviews={reviews} />);
+
+      const review = reviews[0];
```

Comment:
```suggestion
      const [review] = reviews;
```

구조 분해 할당을 사용하면 이렇게 사용할 수 있습니다.

See also
  - https://ko.javascript.info/destructuring-assignment

Reply:
이전에도 구조 분해 할당 하는 방법을 피드백 받았는데 까먹었네요ㅠㅠ
해당 관련된 피드백은 이번이 마지막이라 생각하고 앞으로는 더 신경쓰면서 작업하도록 하겠습니다.

좋은 피드백 감사합니다!!